hub/management/commands/import_mps.py

```diff
@@ -200,3 +210,36 @@ def import_mp_images(self):
             image = File(open(file, "rb"))
             mp.photo.save(f"mp_{mp.external_id}.{extension}", image)
             mp.save()
+
+    def check_for_duplicate_mps(self):
+        areas_covered = list(Person.objects.values_list("area", flat=True))
+        duplicate_areas = [
+            area for area in set(areas_covered) if areas_covered.count(area) > 1
+        ]
+        if duplicate_areas != []:
```

Comment:
FWIW this can be done in one in the database although it's not really worth changing here given how infrequently this runs but this would also work:

``` python
duplicates = (
    Person.objects.distinct()
    .values("area_id")
    .annotate(area_count=Count("area_id"))
    .filter(area_count__gt=1)
)
if duplicates.count() > 0:
```

plus changing `area=area` to `area=area["area_id"]` in the filter further down.

Reply:
This is much cleaner than my implementation, so I've put it in.